File: Python/bee 1013.py
[a, b, c] = map(int, input("").split(" "))

maiorAB = (a+b + abs(a-b))/2
maiorABC = (c+maiorAB + abs(c-maiorAB))/2
print(f"{int(maiorABC)} eh o maior")

File: Ruby/bee 1013.rb
valores = gets.split(" ")
a = valores[0].to_i
b = valores[1].to_i
c = valores[2].to_i

maiorAB = ((a+b+(a-b).abs))/2
maiorABC = ((maiorAB+c+(maiorAB-c).abs))/2

puts "#{maiorABC} eh o maior"
